src/main/java/service/strategy/AddHandler.java

```diff
@@ -0,0 +1,23 @@
+package service.strategy;
+
+import dao.FruitDao;
+import model.Fruit;
+import model.Transaction;
+
+public class AddHandler implements OptionHandler {
+    private static final int DEFAULT_QUALITY = 0;
+    private final FruitDao dao;
+
+    public AddHandler(FruitDao dao) {
+        this.dao = dao;
+    }
+
+    @Override
+    public void apply(Transaction transaction) {
+        Fruit fruit = new Fruit(transaction.getFruitName());
+        int quality = transaction.getQuality();

```

Comment:
```suggestion
        int quantity = transaction.getQuantity();
```
same everywhere in your program